runtimes/huggingface/tests/test_tasks/test_japanese_mask_fill.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from mlserver.settings import ModelSettings, ModelParameters
+from mlserver_huggingface import HuggingFaceRuntime
+from mlserver_huggingface.codecs import HuggingfaceRequestCodec
+
+
+@pytest.fixture
+def bert_japanese_settings(request) -> ModelSettings:
+    if (not request.config.option.test_hg_tasks) or (
+        not request.config.option.test_ja_support
+    ):
+        pytest.skip()
+
+    return ModelSettings(
+        name="model",
+        implementation=HuggingFaceRuntime,
+        parameters=ModelParameters(
+            extra={
+                "task": "fill-mask",
+                "pretrained_model": "cl-tohoku/bert-base-japanese",
```

Comment:
How big is this model, if big is there a smaller model that can be used for testing purposes?

Reply:
I could switch it out for https://huggingface.co/izumi-lab/bert-small-japanese/tree/main which is a smaller model. It's about 70 MB.